runtime/hsa-runtime/core/inc/amd_xdna_driver.h

```diff
@@ -77,12 +77,17 @@ class XdnaDriver final : public core::Driver {
                               void **mem, size_t size,
                               uint32_t node_id) override;
   hsa_status_t FreeMemory(void *mem, size_t size) override;
-
-  /// @brief Creates a context on the AIE device for this queue.
-  /// @param queue Queue whose on-device context is being created.
-  /// @return hsa_status_t
   hsa_status_t CreateQueue(core::Queue &queue) const override;
   hsa_status_t DestroyQueue(core::Queue &queue) const override;
+  hsa_status_t ExportDMABuf(void *va, size_t size, int *dmabuf_fd,
```

Comment:
I used mem throughout. It could be good to use mem for consistency, or update the other functions to use va.